docs/reference/r-system-requirements.md

```diff
@@ -7,16 +7,17 @@ The following are the minimum software and hardware requirements to run Charmed
 
 ### Juju
 
-The charm supports both [Juju 2.9 LTS](https://github.com/juju/juju/releases) and [Juju 3.1](https://github.com/juju/juju/releases).
-
-The minimum supported Juju versions are:
-
-* 2.9.32+ (no tests made for older versions).
-* 3.1.7+ (Juju secrets refactored/stabilized in Juju 3.1.7)
-
-[note type="caution"]
-**Note**: Juju 3.1 is supported from the charm revision 116+
-[/note]
+The charm supports both [Juju 2.9 LTS](https://github.com/juju/juju/releases) and [Juju 3.x](https://github.com/juju/juju/releases) with the following conditions:
+
+| Juju Releases | Juju Supported | Min charm revision |Comment |
```

Comment:
Do we need both the first and second columns for some reason? If we do for some reason, we can at least combine them in a single column by providing version in parenthesis for each line: Juju Release (version).